DXMainClient/DXGUI/Multiplayer/CnCNet/CnCNetLobby.cs

```diff
@@ -1111,14 +1108,25 @@ private void HandleGameInviteCommand(string sender, string argumentsString)
                 return;
             }
 
-            var inviteMessageBox = XNAMessageBox.ShowYesNoDialog(WindowManager, "Invitation received",
-                sender + " wants you to join their game (" + roomName + ").  Accept?");
+            var gameInviteChoiceBox = new ChoiceNotificationBox(WindowManager);
+
+            WindowManager.AddAndInitializeControl(gameInviteChoiceBox);
+
+            // show the invitation at top left for 60 seconds
+            gameInviteChoiceBox.Show(
+                "GAME INVITATION",
+                GetUserTexture(sender),
+                sender,
+                "Join " + gameName + "?",
+                "Yes", "No", 60);
```

Comment:
In my opinion game invitations don't have to expire at all. F. ex. if the user is AFK, he wouldn't even know he was invited in the first place.